tests/e2e/httplog.go

```diff
@@ -306,7 +306,7 @@ func (x *Normalizer) Preprocess(events []*test.LogEntry) {
 			if u != nil {
 				kind := u.PathItems[len(u.PathItems)-1].Resource
 
-				placeholder := x.placeholderForGCPResource(kind)
+				placeholder := x.placeholderForGCPResource(kind, targetId)
```

Comment:
I don't see either this PR or the [tagKey test cases](https://github.com/GoogleCloudPlatform/k8s-config-connector/tree/master/pkg/test/resourcefixture/testdata/basic/tags/v1beta1/tagstagkey) use "namespaced" as the targetId, do you mind sharing more pointers on how this takes affect? my main concern is that the `namespaced` value buried too deep in the code (across 2-3 files) and can make the mockGCP hard to maintain.

Reply:
I added a link when we compare to "namespaced" to the unusual method in question (https://cloud.google.com/resource-manager/reference/rest/v3/tagKeys/getNamespaced)

Reply:
Oh! Fine. that's a weird method. Thanks!